app/views/employees/rewards/orders/new.html.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why did you created method `new` as well as `view` for it if you are not using it anywhere?

Reply:
i was using it in the first place, but then i stopped and never really thought i could delete it